### DISCUSSION TITLE
Sort option

### DISCUSSION THREAD

**Discussion by LiZa111111111** [neperfepx/neper#597] [Q&A] [answered]
Hello,
I would like to use -sort option for sorting the tessellation cells according to their volumes (from smallest to largest) but it is not working. I use following code "neper -T -n 10 -statcell id,vol -sort vol -o cells"
This is what I get in the cells.stcell file:
7 0.111350234620
8 0.121170840144
5 0.091498608845
2 0.070088427857
6 0.093914639734
9 0.129579331792
3 0.079268887665
1 0.067709377224
4 0.080175076329
10 0.155244575789

It does not look sorted.

The purpose of it is to find out cell ids containing largest volumes. 
Is there another way of finding out which grains contains the volumes or seed coordinates within a specific range?
Thanks!

**Comment by rquey** [comment, marked as answer]
It actually *is* sorted. `-sort` acts on the cell ids, not on the cell "positions" (who comes first). See how cell 1 (line 7) has the smallest volume, etc.

I'll make it more clear in the documentation.

**Reply by LiZa111111111** [reply]
Thank you!
But is there a way of how to find the cell ids within the specific range of volumetric or coordinate data?

**Reply by rquey** [reply]
I would do some `awk`:

```
$ neper -T -n 10 -statcell id,vol -sort vol -o cells
$ awk -v min=0.08 -v max=0.12 '{if ($2>min && $2<max) print $1}' cells.stcell 
7
5
6
4
```

You can proceed in the same way for the coordinates.

**Reply by LiZa111111111** [reply]
For instance, if I have 6000 grains, I would like to pick out the grain ids with their seed positions starting from x=<1

**Reply by LiZa111111111** [reply]
Ok. Thank you! That what I meant, yes.
